pkg/rootless/rootless_test.go

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2019 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package rootless
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	//"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+)
+
+func createTestUIDMapFile(input string) error {
+	f, err := os.Create(uidMapPath)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+
+	_, err = f.WriteString(input)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// TestSetRootlessRootlessUID1000 tests that isRootless is set to
+// true when a host UID is not 0
+func TestSetRootlessRootlessUID1000(t *testing.T) {
+	// by default isRootless should be set to false initially
+	assert.False(t, IsRootless())
+
+	uidMapPathStore := uidMapPath
+	uidMapPath = "/tmp/testUIDMapFile"
```

Comment:
It would be safer to avoid hard-coding a path like this as it might already exist. I'd use something like:

```go
assert := assert.New(t)
tmpDir, err := ioutil.TempDir("", "")  
assert.NoError(err)
defer os.RemoveAll(tmpDir)

uidMapPath := filepath.Join(tmpDir, "testUIDMapFile")
```

Reply:
ahh thank you!